### PR TITLE
Add "Alternative Token" handling for flask-login (see #699)

### PR DESCRIPTION
**Pull request by asmith26**

### CODE REVIEW

File: main/auth/auth.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def key(self):
     return self.user_db.key.urlsafe()
 
   def get_id(self):
-    return self.user_db.key.urlsafe()
+    return unicode(self.user_db.session_token)
```
Comment (gmist) [inline main/auth/auth.py]: It seems to me that ```unicode``` call is unnecessary.

``` python
def get_id(self):
  return self.user_db.session_token  # session_token == UUID
```

Comment (asmith26) [inline main/auth/auth.py]: Possibly, I was just filling the [docs](https://flask-login.readthedocs.io/en/latest/#your-user-class):
>get_id()
This method must return a unicode that uniquely identifies this user, and can be used to load the user from the user_loader callback. Note that this must be a unicode - if the ID is natively an int or some other type, you will need to convert it to unicode.

Comment (gmist) [inline main/auth/auth.py]: +1 I really missed it

**Comment by lipis** [comment]
BTW.. is this breaking the backwards compatibility?

**Comment by asmith26** [comment]
Good question, yes it does - sorry I forgot to test that (I'm currently still developing a new app, but will be sure to check this for any future pull requests).

One backwards compatibility problem I see is that existing users won't have been created a (unique) `session_token` (in fact, all existing users will have the same "empty" session token which is definitely not what we won't) - you can find a possible fix in https://github.com/asmith26/gae-init/commit/7d3fa4a98c3f04269ec2e38e6e16c3b6bbc1fbaf

Nonetheless to help keep the code simple, it may be better to hold off this pull request until the next major (e.g. 6.x.x) release? Alternatively, we could label this pull request with [example](https://github.com/gae-init/gae-init/pulls?q=is%3Apr+is%3Aopen+label%3Aexample)?

**Comment by lipis** [comment]
I already merged it though.. maybe we can just include the possible fix (if it actually fixes and not breaking anything) otherwise I will revert this PR.

**Pull request merged**